backends/dpdk/control-plane/bfruntime_ext.cpp

```diff
@@ -89,9 +89,12 @@ void BFRuntimeSchemaGenerator::addActionSelectorGetMemberCommon(
                                     "SelectorGetMember", 1 /* size */, actionSelector.annotations);
 
     auto *keyJson = new Util::JsonArray();
-    addKeyField(keyJson, BF_RT_DATA_SELECTOR_GROUP_ID, "$SELECTOR_GROUP_ID", true /* mandatory */,
-                "Exact", makeType("uint64"));
-    addKeyField(keyJson, BF_RT_DATA_HASH_VALUE, "hash_value", true /* mandatory */, "Exact",
+    addKeyField(keyJson, BF_RT_DATA_SELECTOR_GROUP_ID, "$SELECTOR_GROUP_ID",
```

Comment:
Your code is broken because you also transformed a inline comment here. These comments are parameter hints or function argument comments: 
https://google.github.io/styleguide/cppguide.html#Function_Argument_Comments